tests/app/main/test_services.py

```diff
@@ -303,15 +330,48 @@ def test_should_view_disabled_service_with_removed_message(
             res.get_data(as_text=True)
         )
 
-        self._post_remove_service(service_should_be_modifiable=False)
+    def test_should_not_view_other_suppliers_services(
+            self, data_api_client, supplier_service_editing_fw_params
+    ):
+        framework_slug, framework_name, framework_editable_services = supplier_service_editing_fw_params
+        self.login()
+        self._setup_service(
+            data_api_client, framework_slug, framework_name, service_status='published', service_belongs_to_user=False)
 
+        res = self.client.get('/suppliers/services/123')
+        if not framework_editable_services:
```

Comment:
Ah yes indeed well spotted, I flipped a coin when I came to this one - did I want to add an unnecessary clause or make it less obvious that this test follows the same basic flow as the others. Scanning the code by eye would bring up a "hold on he's missed something here" moment. But looking at it now it does look kinda stupid.

Still, I don't want to have to do a chain-rebase... :hear_no_evil: